content/en/building/reference/api.md

```diff
@@ -505,6 +505,254 @@ Endpoint used by cht-gateway to send sms messages. More documentation in the [ch
+            "created_by_person_uuid": "",
+            "created_by_place_uuid": ""
+        },
+        "reported_date": 1740727312910,
+        "parent": {
+            "_id": "d828971b-d796-45b5-ab1a-943622d906a1",
+            "_rev": "1-1c8e9ae54655e1892686e854e7d404ea",
+            "type": "health_center",
+            "name": "Health Center",
+            "external_id": "GlghB",
+            "notes": "Turbo amor utilis surgo vomica cedo.\nAlveus tabella tondeo itaque.",
+            "meta": {
+                "created_by": "medic",
+                "created_by_person_uuid": "",
+                "created_by_place_uuid": ""
+            },
+            "reported_date": 1742160968002,
+            "parent": {
+                "_id": "3bb8af4d-ee91-4889-9e11-5ba09a833b2e",
+                "_rev": "1-6b109ceccdfcd13f9f78968262d8c82a",
+                "type": "district_hospital",
+                "name": "Hospital",
+                "external_id": "Xge4N",
+                "notes": "Attonbitus sperno cernuus.\nVarius temeritas suadeo cimentarius tum.",
+                "meta": {
+                    "created_by": "medic",
+                    "created_by_person_uuid": "",
+                    "created_by_place_uuid": ""
+                },
+                "reported_date": 1741973509708
+            }
+        }
+    }
+}
+```
+### GET /api/v1/contact/uuid
+
+*Added in 4.18.0*
+
+Returns a JSON array of strings of UUID's of contacts based on the specified page parameters.
+
+#### Permissions
+
+`can_view_contacts`
+
+#### Query Parameters
+
+| Name     | Required | Description                                                                                                                                                                                                                                                                                                                       |
+|----------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| type     | true     | ID of the contact_type for the contacts to fetch. _This field or freetext is required for this endpoint._                                                                                                                                                                                                                         |
+| freetext | true     | A string to search the contacts by. The string value should be atleast 3 in length. The additional condition is that the value should not contain a whitespace(' ') unless the value is in the `key:value` pattern. The `key:value` pattern is for exact match searching. _This field or freetext is required for this endpoint._ |
```

Comment:
suggestion: I think we can follow the pattern from the cht-datasource docs and not mention the `key:value` stuff. (We support that for backwards compatibility, but I would rather not advertise that functionality.)  

> A string search term for filtering the contacts to fetch. The string value should be at least 3 characters in length and not contain whitespace.

Same goes for similar docs below.